tips/TIP-0048/openapi3-core.yaml

```diff
@@ -0,0 +1,3807 @@
+      required:
+        - type
+        - pubKeyHash
+
+    AccountAddress:
+      description: Address of an account output.
+      properties:
+        type:
+          type: integer
+          description: Set to value 8 to denote an Account Address.
+        accountId:
+          type: string
+          description: The hex-encoded, 0x prefixed BLAKE2b-256 hash of the Output ID that created the account.
+      required:
+        - type
+        - accountId
+
+    NFTAddress:
+      description: Address of an NFT account.
+      properties:
+        type:
+          type: integer
+          description: Set to value 16 to denote an NFT Address.
+        nftId:
+          type: string
+          description: The hex-encoded, 0x prefixed BLAKE2b-256 hash of the Output ID that created the NFT.
+      required:
+        - type
+        - nftId
+
+    AddressUnlockCondition:
+      description: Can be unlocked by unlocking the address.
+      properties:
+        type:
+          type: integer
+          description: Set to value 0 to denote an Address Unlock Condition.
+        address:
+          oneOf:
+            - $ref: '#/components/schemas/Ed25519Address'
+            - $ref: '#/components/schemas/AccountAddress'
+            - $ref: '#/components/schemas/NFTAddress'
+      required:
+        - type
+        - address
+
+    ImmutableAccountAddressUnlockCondition:
+      description: Can be unlocked by unlocking the permanent account.
+        The unlock condition has to be kept in future state transitions of the UTXO state machine.
+      properties:
+        type:
+          type: integer
+          description: Set to value 6 to denote an Immutable Account Address Unlock Condition.
+        address:
+          oneOf:
+            - $ref: '#/components/schemas/AccountAddress'
+      required:
+        - type
+        - address
+
+    StorageDepositReturnUnlockCondition:
+      description: Can be unlocked by depositing return amount to return address via an output that only has Address Unlock Condition.
+      properties:
+        type:
+          type: integer
+          description: Set to value 1 to denote a Storage Deposit Return Unlock Condition.
+        returnAddress:
+          oneOf:
+            - $ref: '#/components/schemas/Ed25519Address'
+            - $ref: '#/components/schemas/AccountAddress'
+            - $ref: '#/components/schemas/NFTAddress'
+        amount:
+          type: string
+          description: Amount of IOTA tokens the consuming transaction must deposit to the address defined in Return Address. Plain string encoded number.
+      required:
+        - type
+        - returnAddress
+        - amount
+
+    TimelockUnlockCondition:
+      description: Can be unlocked if the CTT (Confirmed Tangle Time) has a >= Unix Timestamp.
+      properties:
+        type:
+          type: integer
+          description: Set to value 2 to denote a Timelock Unlock Condition.
+        slotIndex:
```

Comment:
This is just `slot` in iota.go, what is correct?

Reply:
`slot` it is.